### PR TITLE
#2002 - VPolytope constructor from matrix columns

### LINKED ISSUE #2002: VPolytope constructor from matrix columns
Extend https://github.com/JuliaReach/LazySets.jl/issues/1987 to higher dimensions, ie. such that the following works: 

```julia
julia> using StaticArrays; VPolytope(@SMatrix(rand(4, 10)))
```

### PR DESCRIPTION
**Pull request by mforets**
Closes #2002.

### CODE REVIEW

File: test/unit_Polytope.jl [hunk 1]
```diff
@@ -246,6 +246,10 @@ for N in [Float64]
     q = VPolytope([N[0, 1], N[0, 2]])
     @test N[0, 1//2] ∉ q
 
+    # creation from a matrix (each column is a vertex)
+    pmat = VPolytope(copy(N[0 0; 1 0; 0 1]'))
```
Comment (schillic) [inline test/unit_Polytope.jl]: Why the `copy` and the transpose?

Comment (mforets) [reply test/unit_Polytope.jl]: Each vertex is a column (hence the transpose); the `copy` is to materialize the lazy transpose, this is standard.

Comment (schillic) [reply test/unit_Polytope.jl]: Okay, I thought that you could as well write the final matrix directly, but if it helps in reading... (for me it does not :wink:)

Comment (mforets) [reply test/unit_Polytope.jl]: This is what comes natural to me, i don't know .. :) I have to think about where to put the commas, if i have to write the final matrix directly.

Comment (mforets) [reply test/unit_Polytope.jl]: You can add a new example in the docstrings if you'd prefer, examples never hurt..

**Pull request merged**